helix-term/src/ui/text_decorations/diagnostics.rs

```diff
@@ -123,6 +123,7 @@ impl Renderer<'_, '_> {
         end_col - start_col
     }
 
+    // need to toggle this
```

Comment:
may I ask what that comment is used for?

Reply:
Oops, looks like a comment I've left for myself while trying to implement this :laughing: